docs/days/day-10.md

```diff
@@ -358,8 +358,9 @@ class CategoryController extends AbstractController
     }
 }
 ```
+You probably have noticed that we do not have to explicitly persist Category object, it is already loaded in EntityManager and the only thing we need to do is to call `flush` method from EntityManager, that will save all changes.  
```

Comment:
`... that will save all changes.  ` - I think it's better to write "... that will apply all changes in database.". I think it would be nice somehow to show that when you call `flush` changes are applied in DB layer.